### DISCUSSION TITLE
Allow mount speed items to work with druid flight form since idol has wotlk stats

### DISCUSSION THREAD

**Discussion by trclocke** [chromiecraft/chromiecraft#5846] [Suggestions]
Normally level 70 druids could get 10% faster flight with the Charm of Swift Flight idol, but it was changed in wotlk to only affect the mana cost. This means druids have no way to increase their flight speed. Allowing items like Riding Crop and Carrot on a Stick to affect flight form would be a nice correction.

**Comment by 55Honey** [comment]
Hello @trclocke ,
Thanks for your suggestion.
The Druid Flight Form is an advantage unique to druids. A druid can still use a regular flying mount to obtain the advantage of the mentioned items. They're not supposed to make use of both advantages simultaneously however.

**Reply by trclocke** [reply]
Thanks for the reply! I mention it because in TBC retail and classic both, druids _were_ supposed to have the option of increased flight form speed through the Charm of Swift Flight idol, part of the swift flight form quest chain. No flight form speed increase is a disadvantage unique to ChromieCraft due to Wrath itemization of TBC items. If druids making use of mount speed items feels unfair, consider reverting the stats on Charm of Swift Flight to what it was during TBC. Just stuff to think about! :)

**Reply by 55Honey** [reply]
If it's wrong for 3.3.5, feel free to open a bug report with a credible source. No need to suggest a change here then ;)

**Reply by trclocke** [reply]
nope! It's correct for 3.3.5, which is why I submitted it as a suggestion instead of a bug. Figured it was worth mentioning since it's available for level 70 druids elsewhere but exclusively not for ChromieCraft druids. Tried to be as clear as I could, let me know if you have questions!

In case it helps, TBC and WOTLK versions, pretty clear why one is preferred :)
https://www.wowhead.com/wotlk/item=32481/charm-of-swift-flight
https://www.wowhead.com/tbc/item=32481/charm-of-swift-flight
